### PR TITLE
Update tooling and READMEs for branch rename

### PR DESCRIPTION
**Pull request by andschwa**
Since `master` is now `main`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ The extension _should_ work anywhere VS Code itself and PowerShell Core 7 or hig
 PowerShell Core 6 is end-of-life and so not supported. Our test matrix includes the
 following:
 
-- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.1
-- **macOS 10.15** with PowerShell Core 7.2.1
-- **Ubuntu 20.04** with PowerShell Core 7.2.1
+- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.4
```
Comment (SeeminglyScience) [inline README.md:31]:
```suggestion
- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.5
```

(unless intentional)

File: README.md [hunk 2]
```diff
@@ -28,9 +28,9 @@ The extension _should_ work anywhere VS Code itself and PowerShell Core 7 or hig
...
+- **macOS 10.15** with PowerShell Core 7.2.5
+- **Ubuntu 20.04** with PowerShell Core 7.2.4
```
Comment (SeeminglyScience) [inline README.md:33]:
```suggestion
- **Ubuntu 20.04** with PowerShell Core 7.2.5
```

(unless intentional)

File: docs/azure_data_studio/README_FOR_MARKETPLACE.md [hunk 3]
```diff
@@ -28,9 +28,9 @@ The extension _should_ work anywhere ADS itself and PowerShell Core 7 or higher
 PowerShell Core 6 is end-of-life and so not supported. Our test matrix includes the
 following:
 
-- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.1.5
-- **macOS 10.15** with PowerShell Core 7.1.5
-- **Ubuntu 20.04** with PowerShell Core 7.1.5
+- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.4
```
Comment (SeeminglyScience) [inline docs/azure_data_studio/README_FOR_MARKETPLACE.md:31]:
```suggestion
- **Windows Server 2019** with Windows PowerShell 5.1 and PowerShell Core 7.2.5
```

(unless intentional)

File: docs/azure_data_studio/README_FOR_MARKETPLACE.md [hunk 4]
```diff
@@ -28,9 +28,9 @@ The extension _should_ work anywhere ADS itself and PowerShell Core 7 or higher
...
+- **macOS 10.15** with PowerShell Core 7.2.5
+- **Ubuntu 20.04** with PowerShell Core 7.2.4
```
Comment (SeeminglyScience) [inline docs/azure_data_studio/README_FOR_MARKETPLACE.md:33]:
```suggestion
- **Ubuntu 20.04** with PowerShell Core 7.2.5
```

(unless intentional)

**Review by SeeminglyScience — CHANGES_REQUESTED** [review]
Just a couple of version number fixes, setting to request changes to avoid auto merge

File: README.md (hunk 1 above)
Comment (andschwa) [reply README.md:31]: Intentional, I checked CI, and didn't want to lie about what it's running _today_.

**Review by SeeminglyScience — APPROVED** [review]
LGTM!

**Pull request merged**